resilience4j-framework-common/src/main/java/io/github/resilience4j/common/ratelimiter/configuration/CompositeRateLimiterCustomizer.java

```diff
@@ -0,0 +1,34 @@
+package io.github.resilience4j.common.ratelimiter.configuration;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * the composite  of any circuit breaker {@link RateLimiterConfigCustomizer} implementations.
+ */
+public class CompositeRateLimiterCustomizer {
+
+    private final Map<String, RateLimiterConfigCustomizer> customizerMap = new HashMap<>();
+
+    public CompositeRateLimiterCustomizer(List<RateLimiterConfigCustomizer> customizers) {
+        if (customizers != null && !customizers.isEmpty()) {
+            customizerMap.putAll(customizers.stream()
+                .collect(
+                    Collectors.toMap(RateLimiterConfigCustomizer::name, Function.identity())));
```

Comment:
Don't we need to check that there are multiple customizer for same name?

Reply:
we can throw an exception to show that is not allowed to have more than one customizer per instance name , is this what u r asking for :) ? or i misunderstood ?

Reply:
Yes right. We can throw an exception or write on docs about supporting just one customizer per instance name.